static/js/pm_conversations.js

```diff
@@ -11,6 +12,20 @@ export function is_partner(user_id) {
     return partners.has(user_id);
 }
 
+function filter_muted_pms(conversation) {
+    // This is used to hide muted users from the top left
+    // corner. We don't hide huddles, since they may include
+    // non-muted users and are more similar to streams.
+    if (
+        !conversation.user_ids_string.includes(",") &&
+        muting.is_user_muted(Number.parseInt(conversation.user_ids_string, 10))
+    ) {
+        return false;
+    }
+
+    return true;
+}
```

Comment:
This feels pretty hacky that we're parsing the `,`s etc., so probably another TODO list item to cleanup after we merge the core behavior.

Reply:
OK that's basically an existing problem with this module; I think what we actually want to do is make `conversation` objects have a `is_group` property, and maybe also a `user_ids` list of integers, that we can check, to have the parsing mostly at the boundaries of this module.

Reply:
I think comma separated user IDs is a convention followed throughout the codebase to denote recipients? Updating this module to use arrays make sense because I think it isn't too tied up with other modules. Shall I add a prep commit here?

Reply:
I think the right way to think about is that `user_ids_string` is a really nice encoding for something we're going to put into a dictionary; and maybe it ends up being the case that for many modules, that's the only place we need to put the set of user IDs.  But in a context where we are going to iterate through the user IDs often, we may want to figure out a way to contain the parsing logic, at least to a function we call in 10 places, or perhaps some other approach.

I don't think doing something here is urgent.